### DISCUSSION TITLE
Welcome to Forza-Horizon-5-Auction-House-Sniper Discussions!

### DISCUSSION THREAD

**Discussion by Scruffydrew** [Scruffydrew/Forza-Horizon-5-Auction-House-Sniper#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by RahndelDaVandal** [comment]
Lol good on you

**Comment by Illuti0n** [comment]
Just wanted to ask, how do you open the program for the FH5 Auction House Sniper? It shows up as a blank file.

**Comment by Scruffydrew** [comment]
when you execute the exe by double clicking on it, it should open a blank window, it should look like this: 
![image](https://user-images.githubusercontent.com/66156463/180671415-4f58515c-9a27-4baf-99b8-bedeb77c2346.png)
you then need to go to your FH5 and after a few seconds the program will start, this delay is there to allow you to get back to the game before it attempts to look for the auction search button.

once the program starts it will display some text on the window it should look like this:
![image](https://user-images.githubusercontent.com/66156463/180671505-579bc1b6-8812-4116-91b8-a2453a2a4f0b.png)

if you need any additional help, please feel free to let me know.

**Reply by Illuti0n** [reply]
Ah, Thank you. I did that, but now when I open it, it says 
![FH5 Error](https://user-images.githubusercontent.com/109923759/180680558-c2db0d99-2a49-4485-9f42-212a755b83b0.PNG)

**Reply by Scruffydrew** [reply]
Currently the exe is in your OneDrive desktop folder,
Try moving the exe to your local desktop then try running it,
let me know how it goes.

**Comment by BlaZe989o** [comment]
When i start the programm nothing happens in the left corner is 'Programm running' and I onened Forzah5 my cursor is in the left corner too but nothing happens

**Reply by BlaZe989o** [reply]
*opened

**Comment by computeur** [comment]
Thank you for the freeware & tutorial

**Comment by computeur** [comment]
does it work with the UWP versiibn (Game pass) ?

**Reply by Scruffydrew** [reply]
i believe it should

**Comment by MozeOfficial** [comment]
Is it Possible that the Program doesnt work anymore ord does it just not work on windows 11?

**Reply by Scruffydrew** [reply]
In theory it shoul still work but I'll test it out on Windows 11 when I find some time but I'm very busy with uni at the moment so it might take some time

**Reply by Scruffydrew** [reply]
ok ive tested it out and it still seems to work on windows 11 but give the latest version a try and see if it works for you

**Comment by mBIMMER** [comment]
that the program no longer works?
i see all stuff (https://user-images.githubusercontent.com/66156463/180671505-579bc1b6-8812-4116-91b8-a2453a2a4f0b.png)  checking if there.....auction
but no automatic clicks to enter and back to auctions

**Reply by Scruffydrew** [reply]
try the previous release, also make sure your brightness is set to the default, and make sure you have a 16:9 resolution set. you also make sure you are clicked into fh5 at all times

**Reply by mBIMMER** [reply]
i try 5 versions and new one
same nothing  (but no automatic clicks to enter and back to auctions)
here some screenshots  https://dropmefiles.com/5E1TI

you mind start game from folder R:\Game\Steam\steamapps\common\ForzaHorizon5  ForzaHorizon5.exe  ?

any help^) pleace

**Reply by Scruffydrew** [reply]
the program can be started from anywhere, it doesnt need to be in the same folder as forza. can you tell me what your settings are set to.
also give this version a go, i just uploaded it https://github.com/Scruffydrew/Forza-Horizon-5-Auction-House-Sniper/releases/tag/v2.0.2

**Reply by mBIMMER** [reply]
thanks now i try
what exactly settings you want?  most setting i show in screenshot previous messag

**Reply by mBIMMER** [reply]
[/v2.0.2](https://github.com/Scruffydrew/Forza-Horizon-5-Auction-House-Sniper/releases/tag/v2.0.2)

same, i see menu and logos
but there is no automatic entry into the auction...

**Reply by Scruffydrew** [reply]
have you clicked back into the game?

**Reply by Scruffydrew** [reply]
if clicking back into the game after the overlay has appeared doesn't work, maybe try setting your game to be 1080p rather than 720p.

**Reply by mBIMMER** [reply]
yes i click on top of welcome text

yes i try various way. and send game to other monitor. and i try various settings.. same result

**Reply by Scruffydrew** [reply]
honestly idk. as far as i can tell it should be working, ive tested it and havent had any issues and i got someone else to test it and they didnt have any issues either

**Comment by ejf9** [comment]
Yea it doesn't work for me either. I tried everything; it gets stuck on:

"Welcome to the Forza Auction House Sniper Bot
Starting program
Forza Horizon 5
Entering search
Line: 66
Checking if there are any cars up for auction"

I am using a laptop with Windows 11 installed. Following the instructions and it's not searching any auctions. I even set it to an auction which has vehicles available, yet it doesn't purchase any.

Not sure what is going on.

**Reply by Scruffydrew** [reply]
whats the resolution of your screen?

**Reply by ejf9** [reply]
resolution 1920x1080, it's weird.. i even disconnected the monitor from my laptop to test it just on the laptop.

**Reply by Scruffydrew** [reply]
give the latest version a try, see if that works for you

**Reply by berzerker629** [reply]
I am having the same issue, same resolution. I have tried adjusting nearly every setting and I seem to keep getting the same thing, 

"Welcome to the Forza Auction House Sniper Bot
Starting program
Forza Horizon 5
Entering search
Line: 66
Checking if there are any cars up for auction"

When I run the newest version, I only get 

"Welcome to the Forza Auction House Sniper Bot
Starting program
Forza Horizon 5
Entering search"

Nothing else seems to happen and nothing changes, no matter the settings I adjust. Any help would be very appreciated, and thank you!

**Comment by StewedLawyer9** [comment]
You are amazing. Thank you.

**Reply by Scruffydrew** [reply]
np

**Reply by StewedLawyer9** [reply]
The bot works really well. The only problem is that sometimes it stutters and clicks the buy out button and the moment it doesn't see the button because it scrolled through it crashes. The plus side is that it is very fast unfortunately it has some bugs. Thanks for your work.

**Comment by StewedLawyer9** [comment]
Is it possible to turn on the bot on the 2nd monitor and use the first monitor without disturbing the program?

**Reply by Scruffydrew** [reply]
In the current state the program is not able to do work on a second monitor, in addition you need to have FH5 constantly set as your current active window, meaning that you would need to be constantly clicked into the game for the program to work. If you would like me to look into and potentially update the program to work on a second monitor then let me know and ill start working on it, although i probably wont start working on it for another couple weeks as after that ill have some free time.

**Reply by StewedLawyer9** [reply]
Thank you for your reply. If you would have the time and desire you could try to make it work on a 2nd monitor. It would be a game changer.

**Comment by Sasocverlin** [comment]
Hey guys has anyone got banned for this?

**Reply by 0XenoX0** [reply]
Does it work for you ?

**Comment by raymondKR** [comment]
Hello, everything is working fine. When the car you want appears in the auction house, press Y. If you press the down key again, you will be taken directly to seller information rather than vehicle purchase. Can you fix it?

**Reply by 0XenoX0** [reply]
Yes i have the exact same problem it's been 2 weeks i am trying to fix this but idk how.

**Comment by TituM-jpeg** [comment]
Hello, how i can speed up a program ? To buy faster

**Comment by TituM-jpeg** [comment]
I found an .exe file somewhere, and it works perfectly with files
downloaded from this site

**Reply by 0XenoX0** [reply]
It's good but i have a problem now that instead of buying the bot shows profile of the player and don't know how to fix it

**Reply by Stashway** [reply]
> It's good but i have a problem now that instead of buying the bot shows profile of the player and don't know how to fix it

Did you fix this ?

**Reply by Stashway** [reply]
> It's good but i have a problem now that instead of buying the bot shows profile of the player and don't know how to fix it

???

**Comment by TituM-jpeg** [comment]
Let me type you later, i hope i have the download file, im gonna send you
with instruction

**Comment by TituM-jpeg** [comment]
For me it works perfectly,
do you do as it says in the instructions?

**Reply by funnyflipp3r** [reply]
My program gets stuck entering search, I feel like I have tried everything how did you get it to work

**Comment by funnyflipp3r** [comment]
> ## 👋 Welcome!
> We’re using Discussions as a place to connect with other members of our community. We hope that you:
> 
> * Ask questions you’re wondering about.
> * Share ideas.
> * Engage with other community members.
> * Welcome others and are open-minded. Remember that this is a community we
>   build together 💪.
> 
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

![Screenshot 2024-09-05 120141](https://github.com/user-attachments/assets/1575e045-d722-4933-bb88-61bee2b26df3)
Gets stuck here

**Reply by funnyflipp3r** [reply]
I've run as an admin and in different aspect ratios, 1920x1080 and 16:9, cant figure out how to make it work

**Comment by Stashway** [comment]
I was able to get one car it’s still working, but it works for like two or three times going in and out and then when the car does pop up, sometimes I’ll buy it out or most of the time go to the game tag and miss The buy button Only issue I’m having is that it misses the buyout button basically

**Comment by TituM-jpeg** [comment]
That is because car was selled already, that happening because of that, if
bot working til that happen, just restart the bot and let him do the work

**Reply by Stashway** [reply]
I don’t think that’s the issue because when the car is sold already, it backs out and tries again. It’s going past the buyout button to the gamer tag info. The last option on the screen.

**Comment by TituM-jpeg** [comment]
I haven't played for a long time, I'll have to look when I have time, maybe
it's the version of the game

**Reply by Stashway** [reply]
OK, thank you just a few new cars came out. I’ve been trying to snipe them and I had no success. I did get one with this Which is actually pretty dope, but it passes the buyout button pretty frequently like every other check

**Comment by TituM-jpeg** [comment]
Yeah, i think the version of game is problem, try to download new version
of program

**Reply by Stashway** [reply]
Yea I’m on 3.0.1

**Reply by Stashway** [reply]
Just re-downloaded it tried it again and the first car I found it went straight down to the last option

**Reply by Stashway** [reply]
![IMG_3298](https://github.com/user-attachments/assets/07452de7-42d9-49b0-9619-0a96492037bd)

**Comment by TituM-jpeg** [comment]
i have to check it out man,
I haven't played for a long time

**Reply by Stashway** [reply]
Ok thanks

**Comment by TituM-jpeg** [comment]
yeah, the program have some problems, have to check my program to see what
is going on

**Reply by Stashway** [reply]
other then that i think its one of best one out ! and if we get this one fix it will be the best one for sure

**Comment by Claiveur** [comment]
![Capture d'écran 2024-11-11 101034](https://github.com/user-attachments/assets/f7e597ba-9fef-47e1-b981-6f73ccd93486)

Yeah, I'm stuck here too, the program doesn't select the right button when a car appears on the AH. Will the dev make a new version of his bot ?

**Reply by Scruffydrew** [reply]
I might have some time over the end of year break to attempt at making an updated version. But that won't be for a few weeks

**Reply by Claiveur** [reply]
Thank you, not being able to use your software makes me miss it a lot

**Reply by Claiveur** [reply]
Hi and happy new year ! Have you had a chance to look into the issue causing the program to bug as expected?

**Comment by RyanB241** [comment]
Just here to confirm this script works, was thinking it was too slow to actually snipe a car but ended up finally getting a 22 superstock
![image](https://github.com/user-attachments/assets/4b47c6f1-266f-487b-b3a1-4edb00697a8f)

**Reply by Claiveur** [reply]
Hi, could you please send me your settings? I am only seeing this issue…

**Comment by Ludicrous15** [comment]
![cmd](https://github.com/user-attachments/assets/ebd364b2-8156-436d-a2d1-c228bb26414f)
![forza](https://github.com/user-attachments/assets/4e8c8f3f-34ea-413b-9467-dc3fe61de400)
Hi, can someone help? I installed everything, opened it, and I get stuck to what you can see in the pictures, it`s turned on but it`s not clicking.  I tried with the game in 1080p 1440p in windowed mode, still no luck, There is that error that you can see in the cmd screenshot, but I don`t know how to get it fixed.

**Reply by Claiveur** [reply]
Hello, if that happens, just open the SEARCH AUCTIONS window

**Reply by Ludicrous15** [reply]
still doesn`t work, I think is because of that error that shows right at the beginning in CMD "C:\Users\KEY\Downloads\Forza-Horizon-5-Auction-House-Sniper-main\Forza-Horizon-5-Auction-House-Sniper-main\Forza_Auction_house_Sniper.py:471: DeprecationWarning: setDaemon() is deprecated, set the daemon attribute instead
  Script.setDaemon(True)"
  
  I don`t know how to get it to go, really struggling here :(

**Reply by maximejfx** [reply]
Disable the HDR ;)

**Comment by DopeZubezi** [comment]
Any way you could maybe make the Purchasing part quicker? I keep missing cars because of it.

**Reply by Ludicrous15** [reply]
How did you make it work for you?

**Reply by DopeZubezi** [reply]
What i did is before i ran the Program, i set everything up first, like the car i wanted and the max buyout price. I then double clicked the program and then right after quickly clicked into forza. One i did that I hit Confirm and backed out of it as if you were sniping manually, and then eventually it'll start sniping for you.

**Comment by ghackt** [comment]
This still works to this day. I wish you had one for buying from advanced search by Username to auto buy friends car to save me time

**Reply by Farider3** [reply]
Can you tell me your ingame Settings? I cant get it past "Entering Search". I tried 1080p, 1440p,set the language to english, ran the exe as Administrator, tried to manully start the auction, reset all the graphics settings to default, tunred off HDR. But it doesnt seem to run. Some Help would be appreciate ^^

**Reply by maximejfx** [reply]
In the terminal, where does it get stuck?

**Comment by fh5sniperguy** [comment]
Hello, the bot is working well but whenever a car comes up it presses the quick buy button but instead of buying out it presses "View Seller"
Any fixes? 
![image](https://github.com/user-attachments/assets/c0e7b15c-51bb-443c-bcb2-dfe9a242eb59)

**Reply by Atlas72d** [reply]
I have the same issue man

**Comment by Atlas72d** [comment]
Yo, so, when a car comes up, it clicks on the wrong button. Has anyone managed to fix that ?

**Comment by Aye-10** [comment]
hey, so when i download the zip i cant seem to find the .exe file. Where is it located / where can i download it?

**Comment by r4fsimons** [comment]
FH6?

**Comment by TimberMeShivers** [comment]
You've successfully ruined the art of auction house sniping. Thanks a million!
